java/test/jmri/jmrix/AbstractNetworkPortControllerTestBase.java

```diff
@@ -6,4 +6,8 @@
  * @author      Paul Bender Copyright (C) 2016
  */
 public abstract class AbstractNetworkPortControllerTestBase extends AbstractPortControllerTestBase {
+
+    public void testGetAndSetOutputInterval() {
```

Comment:
Having this empty method here is a little confusing.  It is overriding a test in the base class, but it doesn’t do anything.

Please add an override and a test annotation and a comment in the body explaining why this is empty.

Reply:
I took out the method here (and in ``AbstractSerialPortControllerTestBase`` as per your next comment). Just run the test in specific classes.